src/cryptography/hazmat/backends/openssl/ocsp.py

```diff
@@ -95,6 +95,10 @@ def serial_number(self):
     def hash_algorithm(self):
         return _hash_algorithm(self._backend, self._cert_id)
 
+    @property
```

Comment:
Should this be a `@cached_property`? IIRC the other extensions are

Reply:
Sure, updating :)